### PR TITLE
Ft 104 multi file sync handle set deletions

### PR DESCRIPTION
**Pull request by swordEdge**

### CODE REVIEW

File: src/storage/schemas/multiFileSchema.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import z from 'zod';
 import { TokenSetStatus } from '@/constants/TokenSetStatus';
-import { singleTokenSchema } from './singleTokenSchema';
+import { tokensMapSchema } from './tokensMapSchema';
 
-export const multiFileSchema = z.record(singleTokenSchema).or(z.array(z.object({
+export const multiFileSchema = z.record(tokensMapSchema).or(z.array(z.object({
```
Comment (LiamMartens) [inline src/storage/schemas/multiFileSchema.ts:5]:
```suggestion
export const multiFileSchema = tokensMapSchema.or(z.array(z.object({
```

**Review by LiamMartens — CHANGES_REQUESTED** [review]

**Comment by LiamMartens** [comment]
@swordEdge are we able to update the test coverage to pass?

**Comment by swordEdge** [comment]
> @swordEdge are we able to update the test coverage to pass?

Modified test coverage for multi-file sync. I think it's still failing because of another unit test.

**Comment by LiamMartens** [comment]
@swordEdge it's stil failing because `src/storage/GithubTokenStorage.ts` has a branch coverage diff of `-1`, maybe you can check which lines can be covered additionally?

**Pull request merged**